### DISCUSSION TITLE
Is model.predict() the solution ?

### DISCUSSION THREAD

**Discussion by athtareq** [lululxvi/deepxde#676] [Q&A] [answered]
Hello, how do you call the solution after the training is done ? Because when I use, say, model.compile(3) I get `Cannot feed value of shape () for Tensor 'Placeholder_261:0', which has shape '(?, 1)'
`

Thank you

**Comment by lululxvi** [comment]
You can use `Y = model.predict(X)`, where `X` is a N by d matrix.

**Comment by athtareq** [comment]
Thanks for the quick answer, but can you please give me a quick example ? Sorry and thanks !

**Comment by lululxvi** [comment]
Check the example: https://github.com/lululxvi/deepxde/blob/master/examples/Burgers.py

**Comment by athtareq** [comment]
But for one single point ? e.g y is the solution, how to write y(3) ? Again, sorry for the noobish questions

**Comment by lululxvi** [comment, marked as answer]
Then use `X = np.array([[3]])`.

**Comment by athtareq** [comment]
Thank you very much, you're a godsend !
